pkg/manager/internal.go

```diff
@@ -130,7 +130,7 @@ type controllerManager struct {
 	// election was configured.
 	elected chan struct{}
 
-	webhookServer *webhook.Server
+	webhookServer webhook.Server
```

Comment:
What are some use cases that require an interface instead?

Reply:
Basically anything where the default implementation isn't working. The case I have at hand is that the same binary also generates the certificates, so `Start` has to wait until they exist.